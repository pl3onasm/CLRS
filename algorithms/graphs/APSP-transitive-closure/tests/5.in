a g 
c f 
f a 
b a 
d b 
b e 
e d 
d g 
d c
