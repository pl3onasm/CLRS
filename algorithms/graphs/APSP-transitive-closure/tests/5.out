
--------------------
 GRAPH
 Directed
 Unweighted
 Vertices: 7
 Edges: 9
--------------------
  a[1]: g
  b[2]: a, e
  c[1]: f
  d[3]: g, c, b
  e[1]: d
  f[1]: a
  g[0]
--------------------


--------------------
 Initial matrix
--------------------

  1  0  0  0  0  0  1
  1  1  0  0  1  0  0
  0  0  1  0  0  1  0
  0  1  1  1  0  0  1
  0  0  0  1  1  0  0
  1  0  0  0  0  1  0
  0  0  0  0  0  0  1

--------------------


--------------------
 Transitive closure
--------------------

  1  0  0  0  0  0  1
  1  1  1  1  1  1  1
  1  0  1  0  0  1  1
  1  1  1  1  1  1  1
  1  1  1  1  1  1  1
  1  0  0  0  0  1  1
  0  0  0  0  0  0  1

--------------------
